test/styleguide/legacy_bridge.js

```diff
@@ -10,12 +10,36 @@
 window.PROTO_ROOT = '/protos';
 window.nodeSetImmediate = () => {};
 
+window.libphonenumber = {
+  parse: number => ({
+    e164: number,
+    isValidNumber: true,
+    getCountryCode: () => '1',
+    getNationalNumber: () => number,
+  }),
+  isValidNumber: () => true,
+  getRegionCodeForNumber: () => '1',
+  format: number => number.e164,
+  PhoneNumberFormat: {},
+};
+
 window.Signal = {};
+window.Signal.HTML = {
+  render: thing => thing,
+};
 window.Signal.Backup = {};
 window.Signal.Crypto = {};
 window.Signal.Logs = {};
 window.Signal.Migrations = {
-  getPlaceholderMigrations: () => {},
+  getPlaceholderMigrations: () => [{
+    migrate: (transaction, next) => {
+      console.log('migration version 1');
+      transaction.db.createObjectStore('conversations');
+      next();
+    },
+    version: 1,
+  }],
```

Comment:
Maybe we could run migrations explicitly for the style guide at some point 😄

Reply:
No, I really want to move away from this. I want this to cause us pain until we finally uncouple our views from the database. Really a bit disturbing even to type that phrase out... :0/